### ISSUE TITLE
Satoshi address

### ISSUE THREAD

**Issue by futuremoneybtc** [JeanLucPons/Kangaroo#34]
What do you think guys some one is guess this [17XiVVooLcdCUCMf9s4t4jTExacxwFS5uh](https://www.blockchain.com/btc/address/17XiVVooLcdCUCMf9s4t4jTExacxwFS5uh) address.. i am pretty excited to know the bit range of those addresses. I hear that satoshi lost the private key of those addresses, is it true ???

**Comment by dem10** [comment]
Very strange, but even the public key is not tracked.
https://blockchain.info/q/pubkeyaddr/17XiVVooLcdCUCMf9s4t4jTExacxwFS5uh

**Comment by SatoshiNakamotoBitcoins** [comment]
@futuremoneybtc The bitrange is N so FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141

Happy hunting!

Cheers Satoshi

**Comment by Shadow145-cpu** [comment]
@SatoshiNakamotoBitcoins  could I know what program / scrypt to use to calculate range?

**Comment by SatoshiNakamotoBitcoins** [comment]
@Shadow145-cpu No...it is not possible to calculate the range of a BTC address (only when it is given like the puzzles)

**Comment by Shadow145-cpu** [comment]
> @Shadow145-cpu No...it is not possible to calculate the range of a BTC address (only when it is given like the puzzles)

@SatoshiNakamotoBitcoins  I understand.  How can I get the public key of the btc address?

**Comment by hamnaz** [comment]
all satoshi addresses have pubkey :)

**Comment by SatoshiNakamotoBitcoins** [comment]
@Shadow145-cpu You can find them with the blockchain...for example this address...click on the hash where the 50 BTC are mined...with "output" you need the "pkscript" that is the pubkey...(back in the days the pubkey was exposed with mining, after a few years the code has changed a bit...and the pubkey came only exposed with a spent transaction)
![1](https://user-images.githubusercontent.com/45945164/84564422-d24e4800-ad61-11ea-9e94-2dd8a4b097a9.PNG)

**Comment by SatoshiNakamotoBitcoins** [comment]
@hamnaz Yes it is...;-)

**Comment by Shadow145-cpu** [comment]
> @Shadow145-cpu You can find them with the blockchain...for example this address...click on the hash where the 50 BTC are mined...with "output" you need the "pkscript" that is the pubkey...(back in the days the pubkey was exposed with mining, after a few years the code has changed a bit...and the pubkey came only exposed with a spent transaction)
> ![1](https://user-images.githubusercontent.com/45945164/84564422-d24e4800-ad61-11ea-9e94-2dd8a4b097a9.PNG)

@SatoshiNakamotoBitcoins  I know it :) I mean something more automated :) I am looking for some script that will save the public key with a positive balance to the file.

**Comment by dem10** [comment]
> I know it :) I mean something more automated :) I am looking for some script that will save the public key with to the file.

To do this, you will have to scan the entire blockchain. There must be a python tool on the github.

**Comment by SatoshiNakamotoBitcoins** [comment]
@Shadow145-cpu With blockchain.com there are several API's...you have to play around with something like this... https://www.blockchain.com/api/q and https://blockchain.info/q/pubkeyaddr
replace in the end with "pubkeyaddr" for the BTC address to search for...do some linux magic...e.g. wget https://blockchain.info/q/pubkeyaddr...copy the lines and add in the end all the BTC addresses you want to search for and tadaa...(spoiler alert...for your and our unconvenience the old satoshi wallets can not be retrieved like this with Blockchain.com, so that means manual search work...;-))

Nevertheless a list of pubkeys in this moment will bring your nothing because there is no tool out there to do a multi pub key attack...

**Comment by Shadow145-cpu** [comment]
@SatoshiNakamotoBitcoins  thank you :)

**Comment by Shadow145-cpu** [comment]
> @Shadow145-cpu With blockchain.com there are several API's...you have to play around with something like this... https://www.blockchain.com/api/q and https://blockchain.info/q/pubkeyaddr
> add in the end for the BTC address to search for like this https://blockchain.info/q/pubkeyaddr/12JzYkkN76xkwvcPT6AWKZtGX6w2LAgsJg ...do some linux magic...e.g. wget https://blockchain.info/q/pubkeyaddr...copy the lines and add in the end all the BTC addresses you want to search for and tadaa...(spoiler alert...for your and our unconvenience the old satoshi wallets can not be retrieved like this with Blockchain.com, so that means manual search work...;-))
> 
> Nevertheless a list of pubkeys in this moment will bring your nothing because there is no tool out there to do a multi pub key attack...

@SatoshiNakamotoBitcoins  thank you :)

**Comment by Vovkaruss** [comment]
> @Shadow145-cpu You can find them with the blockchain...for example this address...click on the hash where the 50 BTC are mined...with "output" you need the "pkscript" that is the pubkey...(back in the days the pubkey was exposed with mining, after a few years the code has changed a bit...and the pubkey came only exposed with a spent transaction)
> ![1](https://user-images.githubusercontent.com/45945164/84564422-d24e4800-ad61-11ea-9e94-2dd8a4b097a9.PNG)

What do you think, is the statement true that the program did not miss, for some reason, the waste of the first block of coins from this well-known address? -
1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa

**Comment by futuremoneybtc** [comment]
I read somewhere that old bitcoin addresses have 160 bits private keys.. is it true or I have the wrong information?

**Comment by SatoshiNakamotoBitcoins** [comment]
@Vovkaruss What do you actually mean with the "waste" of the first block??

**Comment by SatoshiNakamotoBitcoins** [comment]
@futuremoneybtc Yes I heard that rumour too...but if it is true...we will know when we crack it...;-)

**Comment by Vovkaruss** [comment]
> @Vovkaruss What do you actually mean with the "waste" of the first block??

I apologize for the incorrect transcription. I did not mean the waste of the first block, but the unused output of the transaction of new coins from the zero genesis of the block. I read somewhere that the coins obtained in the zero block cannot be spent due to the peculiarities of the bitcoin core software? Thanks for the answer!

**Comment by SatoshiNakamotoBitcoins** [comment]
@Vovkaruss No need for an apology...when you are able to find the private key to the public of that BTC address you will we able to move the funds...only the dust which is sended over the years to this address wil "track" you down all over the world...;-)

Happy hunting!!

Cheers Satoshi
